src/components/LogoutTest.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { instanceOf } from 'prop-types';
+import { logout, useNavigate } from '../utils/auth_utils';
+import { clearCookies, Cookies, withCookies } from '../utils/cookie_utils';
+
+const LogoutTest = ({ cookies }) => {
+  const navigate = useNavigate();
+  const handleSubmit = async () => {
+    await logout();
+    navigate('/');
+    clearCookies(cookies);
```

Comment:
similar to previous comments, `logout` should include a parameter for redirected route after success. also move logic for clearing cookies to `logout` function as well. also add try/catch around `logout`